crates/wasmtime/src/store.rs

```diff
@@ -966,6 +966,12 @@ impl InterruptHandle {
     }
 }
 
+impl std::fmt::Debug for InterruptHandle {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("InterruptHandle").finish()
+    }
+}
```

Comment:
`VMInterrupts` implements `Debug`, so you should be able to derive `Debug` for `InterruptHandle`.

Reply:
`VMInterrupts` is a private imiplementation detail though, with no way to access it via the handle.

I feel including it in the debug output would be more confusing than helpful.

Reply:
Maybe you could at least expose the `Arc` pointer? That would allow for distinguishing between multiple `InterruptHandle` instances.

Reply:
In that case it seems more sensible to just derive it.

Pushed an update.